components/companycam/sources/new-document/new-document.mjs

```diff
@@ -0,0 +1,28 @@
+import common from "../common/webhook.mjs";
+import scopes from "../common/scopes.mjs";
+
+export default {
+  ...common,
+  key: "companycam-new-document",
+  name: "New Document",
```

Comment:
According to the [Contribution Guidelines](https://pipedream.com/docs/components/guidelines/#general), "Source keys should use past tense verbs that describe the event that occurred (e.g., linear_app-issue-created-instant)". Also, webhook sources should have "(Instant)" at the end of the name. This applies to all the CompanyCam sources.
```suggestion
  key: "companycam-new-document-created",
  name: "New Document Created (Instant)",
```